### PR TITLE
Update praekeltfoundation/gluster to 0.2.0

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: puppet/manifests/default.pp [hunk 1]
```diff
@@ -6,17 +6,8 @@
...
-    ensure  => '3.7.6*',
-    require => [Apt::Ppa['ppa:gluster/glusterfs-3.7'], Class['apt::update']],
-  }
-
-  service { 'glusterfs-server':
-    ensure  => 'running',
-    require => Package['glusterfs-server'],
-  }
+  # The default repo and version are suitable for our needs.
+  class { 'gluster': }
```
Comment (JayH5) [inline puppet/manifests/default.pp]: I prefer the `include gluster` syntax.

Comment (jerith) [inline puppet/manifests/default.pp]: Good point, thanks.

File: puppet/.puppet-lint.rc [hunk 2]
```diff
@@ -0,0 +1 @@
+--no-autoloader_layout
```
Comment (JayH5) [inline puppet/.puppet-lint.rc:3]: I think we should have all the puppet-lint flags either in `.puppet-lint.rc` or the `Rakefile`.. so maybe remove the flags from the Rakefile?

**Comment by JayH5** [comment]
:+1:

**Pull request merged**